tuned_lens/load_artifacts.py

```diff
@@ -14,6 +14,7 @@ def load_lens_artifacts(
     config_file: str = "config.json",
     ckpt_file: str = "params.pt",
     subfolder: str = "lens",
+    cache_dir: Optional[str] = None,
 ) -> Tuple[Path, Path]:
```

Comment:
nit: we can use `tuple[Path, Path]` instead of `typing.Tuple` since we don't support Python versions below 3.9